src/ucm/util/replace.c

```diff
@@ -34,37 +38,77 @@ UCM_DEFINE_REPLACE_FUNC(shmdt,   int,   -1,         const void*)
 UCM_DEFINE_REPLACE_FUNC(sbrk,    void*, MAP_FAILED, intptr_t)
 UCM_DEFINE_REPLACE_FUNC(madvise, int,   -1,         void*, size_t, int)
 
+#if HAVE_DECL_SYS_MMAP
```

Comment:
in case if corresponding syscall is defined - both methods of call original routine are supported (switched via runtime parameter), else only overridden call is supported